### ISSUE TITLE
storage: ObjectHandle.Update does not delete ContentEncoding

### ISSUE THREAD

**Issue by drtriumph** [GoogleCloudPlatform/google-cloud-go#694]
I'm running into an issue where the ObjectHandle.Update function is not updating the ContentEncoding of an object. The [docs](https://godoc.org/cloud.google.com/go/storage#ObjectAttrsToUpdate) suggest that in order to remove the ContentEncoding header from an object, you can write something like this:

```go
// Change only the content encoding of the object.
objAttrs, err := client.Bucket("my-bucket").Object("my-object").Update(ctx, storage.ObjectAttrsToUpdate{
	ContentEncoding: "",
})
```

But this seems to leave the ContentEncoding untouched. Setting the value to anything other than the empty string (`"gzip"`, `"foo"`, anything really) seems to work just fine, but the empty string does nothing

**Comment by drtriumph** [comment]
Looks like it's a copy/paste issue on this line: https://github.com/GoogleCloudPlatform/google-cloud-go/blob/master/storage/storage.go#L363 - instead it should be

```go
forceSendFields = append(forceSendFields, "ContentEncoding")
```

**Comment by bradfitz** [comment]
@drtriumph, yes, that sure looks like a copy/pasteo.

/cc @adams-sarah

**Comment by drtriumph** [comment]
https://code-review.googlesource.com/#/c/14550/

**Comment by drtriumph** [comment]
Closed by 95d975824b489625799f6832e2027b2699d9c88d
